crappydesign_scraped02.json

```diff
@@ -1 +1,102 @@
-[{"num_comments": 64, "title": "Team Taco Bell!"}, {"num_comments": 14, "title": "Anyone up for a BBQ? May as well make the most of it!"}, {"num_comments": 13, "title": "These wall hooks."}, {"num_comments": 10, "title": "What's Wikipedia trying to tell us?"}, {"num_comments": 983, "title": "For when you want a near death experience every time you use the stairs"}, {"num_comments": 10, "title": "Turn the thing in one of the directions depending on the language you speak. In the end only one of the two is correct - guess which one \ud83e\udd14"}, {"num_comments": 8, "title": "She is fading into the ether!"}, {"num_comments": 10, "title": "Is this how an acronym works?"}, {"num_comments": 17, "title": "Crappy design indeed"}, {"num_comments": 54, "title": "I'd rather not..."}, {"num_comments": 6, "title": "Is she hiding something between legs?"}, {"num_comments": 0, "title": "\"Popcorn in 4 easy steps\", you say?"}, {"num_comments": 18, "title": "Words to live by"}, {"num_comments": 8, "title": "Took me a minute..."}, {"num_comments": 4, "title": "It goes to the hot water heater outside the house. Took me months to figure it out."}, {"num_comments": 0, "title": "There was an attempt."}, {"num_comments": 41, "title": "Evil! Jew! Poor man!"}, {"num_comments": 19, "title": "What a pattern!"}, {"num_comments": 33, "title": "This AP Physics practice test has two answers that mean the same thing"}, {"num_comments": 2, "title": "I think I'll take my chances at another hospital."}, {"num_comments": 2, "title": "For the College of Design, it's kinda crappy"}, {"num_comments": 2, "title": "Maybe just have one question asking for your age or age range?"}, {"num_comments": 4, "title": "This metal ring blocking the actual steps, with crappier steps installed to the right for people to go around it"}, {"num_comments": 13, "title": "This light turns green when the train is coming"}, {"num_comments": 2, "title": "But... What does D. A. B. stand for?"}]
\ No newline at end of file
+[
```

Comment:
It's always a good idea to reformat your json, so you can better see what kind of structure it is in, and which keys are being used.